phoebe/frontend/bundle.py

```diff
@@ -3765,11 +3767,12 @@ def run_checks_compute(self, compute=None, solver=None, solution=None, figure=No
 
             pb_needs_Inorm = True
             pb_needs_Imu = True
-            pb_needs_ld = True #np.any([p.get_value()!='interp' for p in self.filter(qualifier='ld_mode', dataset=pbparam.dataset, context='dataset', **_skip_filter_checks).to_list()])
+            pb_needs_ld = True
             pb_needs_ldint = True
 
             missing_pb_content = []
 
+            # TODO: do we really want to run this every time we run checks? Perhaps move to init?
```

Comment:
yes, because the selected passband is allowed to change.  We could probably do some smarter caching and only checking changed parameters, etc, but that would require quite a bit of refactoring.